src/Silverback.Integration.Kafka/Messaging/Broker/InnerConsumerWrapper.cs

```diff
@@ -175,11 +175,10 @@ private void InitInnerConsumer()
 
                     CreateScopeAndPublishEvent(new KafkaErrorEvent(error));
                 })
-                .SetStatisticsHandler((_, json) =>
+                .SetStatisticsHandler((_, statistics) =>
                 {
-                    _logger.LogInformation($"Statistics: {json}");
-
-                    CreateScopeAndPublishEvent(new KafkaStatisticsEvent(json));
+                    _logger.LogDebug($"Statistics: {statistics}");
```

Comment:
I don't like that the statistics clutter my log. Might we can log an information "received statistics from consumer xy or producer xy". When some one turns on the statistics, I assume that they want to do something with them and therefore it is not necessary to log them as information. Wdyt?

Furthermore, I would like to discuss to expose .NET EventCounters based on the statistics, then log them as information makes even less sense.

Reply:
Lowering the log level makes sense. Thanks.